docs/guide/theme-nav.md

```diff
@@ -132,6 +132,26 @@ export default {
 }
 ```
 
+### Hide navbar
+
+It is possible to hide the navbar globally. This can be achived by setting the `showNav` attribute to false. The navbar will always be shown by default.
+
+```js
+export default {
+  themeConfig: {
+    showNav: false
+  }
+}
+```
+
+There is also the possibility to hide the navbar for specific pages. This can be done by declaring `nav: false` in the frontmatter.
+
+```md
+---
+nav: false
```

Comment:
`themeConfig.showNav` and `frontmatter.nav` are inconsistent and confusing

Reply:
I thought about that while writing the code. But I couldn't use _nav_ because it's already used for the nav entries itself in the theme config. So I ended up with two variations since _nav_ in formatter was originally requested in #1227. But now I choose _showNav_ for both.